sdk/tests/client/input_selection/foundry_outputs.rs

```diff
@@ -72,14 +80,28 @@ fn existing_input_alias_for_foundry_alias() {
         None,
         None,
     )]);
-    let outputs = build_outputs(vec![Foundry(
+    let mut outputs = build_outputs(vec![Foundry(
         1_000_000,
         alias_id_2,
-        0,
+        1,
         SimpleTokenScheme::new(U256::from(0), U256::from(0), U256::from(10)).unwrap(),
         None,
     )]);
 
+    outputs.push(
+        AliasOutputBuilder::new_with_amount(251_500, alias_id_2)
+            .add_unlock_condition(StateControllerAddressUnlockCondition::new(
+                Address::try_from_bech32(BECH32_ADDRESS_ED25519_0).unwrap(),
+            ))
+            .add_unlock_condition(GovernorAddressUnlockCondition::new(
+                Address::try_from_bech32(BECH32_ADDRESS_ED25519_0).unwrap(),
+            ))
+            .with_state_index(1)
+            .with_foundry_counter(1)
+            .finish_output(protocol_parameters.token_supply())
+            .unwrap(),
+    );
```

Comment:
What's that for? Should work fine without? Isn't it even better without so it's supposed to pick the right values?

Reply:
Oh is that because of the foundry counter that we can't set with the enum?

Reply:
I added that  before I automatically updated the state index, undid the change now :+1: